tests/unit/mixins/FluxibleMixin.js

```diff
@@ -54,6 +127,10 @@ describe('StoreListenerMixin', function () {
                 statics: {
                     storeListeners: [MockStore]
                 },
+                childContextTypes: {
```

Comment:
Shouldn't the FluxibleMixin provide this already? Why does it need to be declared in every test component?